pages/editAccount.tsx

```diff
@@ -16,6 +14,9 @@ export default function EditAccount() {
     const [showPassword2, setShowPassword2] = useState(false);
     const [showPassword3, setShowPassword3] = useState(false);
     const [users, setUsers] = useState();
+    let name1:string="";
```

Comment:
Always use `useState()` when you want the variable show on HTML and change the value as needed